db/schema.rb

```diff
@@ -144,9 +136,9 @@
     t.integer "form_id", limit: 4, null: false
     t.text "group_hint_translations", limit: 65535
     t.text "group_name_translations", limit: 65535
-    t.integer "group_rank", limit: 4
```

Comment:
Any idea what this was used for and why it isn't anymore? Appreciate the cleanup, just making sure there's no chance this is called by some metaprogramming somewhere that makes it hard to search.